### PR TITLE
Fix improve mode

### PR DESCRIPTION
**Pull request by skinkie**
Fix #376 (directly on master)

@Aurige would this be ok?

### CODE REVIEW

File: xsd/netex_framework/netex_genericFramework/netex_place_version.xsd [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <xsd:schema xmlns="http://www.netex.org.uk/netex" xmlns:netex="http://www.netex.org.uk/netex" xmlns:siri="http://www.siri.org.uk/siri" xmlns:xsd="http://www.w3.org/2001/XMLSchema" targetNamespace="http://www.netex.org.uk/netex" elementFormDefault="qualified" attributeFormDefault="unqualified" version="1.1" id="netex_place_version">
 	<xsd:include schemaLocation="netex_place_support.xsd"/>
 	<xsd:include schemaLocation="../netex_genericFramework/netex_zoneProjection_version.xsd"/>
+	<xsd:include schemaLocation="../netex_reusableComponents/netex_mode_support.xsd"/>
```
Comment (skinkie) [inline xsd/netex_framework/netex_genericFramework/netex_place_version.xsd:5]: @ue71603 @Aurige when I remove this include, processing still works. The AccessModeEnumeration is only mentioned in netex_mode_support, so I am technically puzzled how it "correctly" ends up there.

Comment (Aurige) [reply xsd/netex_framework/netex_genericFramework/netex_place_version.xsd:5]: I checked here: it does not validate as a stand-alone file if you remove this include, it works only when included by others who have already included netex_mode_support.xsd ... so it is best keeping the include here

**Review by Aurige — APPROVED** [review]

File: xsd/netex_framework/netex_reusableComponents/netex_mode_support.xsd [hunk 2]
```diff
@@ -68,19 +68,23 @@ Rail transport, Roads and Road transport
 	<!--======Out of Vehicle==================================================================================-->
 	<xsd:element name="AccessMode" type="AccessModeEnumeration">
 		<xsd:annotation>
-			<xsd:documentation>Access MODE for SITEs.</xsd:documentation>
+			<xsd:documentation>Access MODE for SITEs and VEHICLE MEETING POINTs. Also used for TRANSFER and if OTHER LEG for the Trip model a monomodal non-public transport is used.</xsd:documentation>
```
Comment (trurlurl) [inline xsd/netex_framework/netex_reusableComponents/netex_mode_support.xsd:71]:
```suggestion
			<xsd:documentation>Access MODE for SITEs and VEHICLE MEETING POINTs. Also used for TRANSFER and if in OTHER LEG of the TRIP a monomodal non-public transport is used.</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_mode_support.xsd [hunk 3]
```diff
@@ -210,36 +214,98 @@ Rail transport, Roads and Road transport
...
-			<xsd:enumeration value="ferry"/>
-			<xsd:enumeration value="lift"/>
+			<xsd:enumeration value="ferry">
+				<xsd:annotation>
+					<xsd:documentation>Ferry can be passenger ferries and/or car ferries. The specialisation from water is the detailed schedule and usually the very rigid pattern with only a few stops on the route.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="lift">
+				<xsd:annotation>
+					<xsd:documentation>General Mode for Modes moved by cable. Especially lifts are not only vertical elevators. If a better specialisation applies (like cableway) use that.</xsd:documentation>
```
Comment (trurlurl) [inline xsd/netex_framework/netex_reusableComponents/netex_mode_support.xsd:291]: I guess this should also include all sorts of elevators? There exist hydraulic and cogwheel-driven elevators. Proposal: "Mode for elevators and for modes moved by cable...."

Comment (skinkie) [reply xsd/netex_framework/netex_reusableComponents/netex_mode_support.xsd:291]: Sure, make it a suggestion.

Comment (trurlurl) [inline xsd/netex_framework/netex_reusableComponents/netex_mode_support.xsd:291]:
```suggestion
					<xsd:documentation>General Mode for elevators and for Modes moved by cable. Especially lifts are not only vertical elevators. If a better specialisation applies (like cableway) use that.</xsd:documentation>
```

**Review by trurlurl — APPROVED** [review]

File: xsd/netex_framework/netex_reusableComponents/netex_mode_support.xsd (hunk 3 above)
Comment (Aurige) [reply xsd/netex_framework/netex_reusableComponents/netex_mode_support.xsd]: I would rather go to the Lift Equipment description to go to that level of details (modes are expected to be generic)

Comment (ue71603) [reply xsd/netex_framework/netex_reusableComponents/netex_mode_support.xsd]: @trurlurl  can you accept Christophe's comment and we don't change it here?

Comment (ue71603) [reply xsd/netex_framework/netex_reusableComponents/netex_mode_support.xsd]: @trurlurl sorry is merged already. so done.

**Pull request merged**